apps/wpcom-block-editor/src/utils.js

```diff
@@ -29,10 +29,33 @@ export function sendMessage( message ) {
 }
 
 /**
- * Indicates if the block editor has been initialized with blocks.
+ * Indicates if the block editor has been initialized.
  *
  * @returns {Promise} Promise that resolves when the editor has been initialized.
  */
+export const isEditorReady = async () =>
+	new Promise( ( resolve ) => {
+		const unsubscribe = subscribe( () => {
+			// Calypso sends the message as soon as the iframe is loaded, so we
+			// need to be sure that the editor is initialized and the core blocks
+			// registered. There is an unstable selector for that, so we use
+			// `isCleanNewPost` otherwise which is triggered when everything is
+			// initialized if the post is new.
+			const editorIsReady = select( 'core/editor' ).__unstableIsEditorReady
```

Comment:
This check comes from https://github.com/Automattic/wp-calypso/blob/3a43eac69ebda8790ce75f8b0ed6aec2018f8a56/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L139